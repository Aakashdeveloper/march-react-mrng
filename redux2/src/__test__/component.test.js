import React from 'react';
import Footer from '../component/Footer';
import {create} from 'react-test-renderer';

describe('SnapShot Footer test',() => {
    test('testing Footer',() => {
        let tree = create(<Footer/>);
        expect(tree.toJSON()).toMatchSnapshot()
    })
})