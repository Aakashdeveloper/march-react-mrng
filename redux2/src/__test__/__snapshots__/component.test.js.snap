// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapShot Footer test testing Footer 1`] = `
<footer>
  <div>
    <hr />
    Â© Developer Funnel
  </div>
</footer>
`;
